src/__tests__/Search/SearchView.test.tsx

```diff
@@ -1,21 +1,30 @@
 import { renderHook } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
-import { act } from 'react-dom/test-utils';
 
 import SearchView from '../../components/Search/SearchView';
 import { Strings } from '../../resources/Strings';
 import useProtocolTheme from '../../theme/protocolTheme';
-import { RevisionsList, InputType } from '../../types/state';
 import getTestData from '../utils/fixtures';
-import { renderWithRouter, store } from '../utils/setupTests';
-import { screen, waitFor } from '../utils/test-utils';
+import { renderWithRouter } from '../utils/setupTests';
+import { screen, waitFor, act, FetchMockSandbox } from '../utils/test-utils';
 
 const protocolTheme = renderHook(() => useProtocolTheme()).result.current
   .protocolTheme;
-
 const toggleColorMode = renderHook(() => useProtocolTheme()).result.current
   .toggleColorMode;
+
+function setupTestData() {
+  const { testData } = getTestData();
+  (global.fetch as FetchMockSandbox).get(
+    'begin:https://treeherder.mozilla.org/api/project/try/push/',
+    {
+      results: testData,
+    },
+  );
+}
+
 function renderComponent() {
+  setupTestData();
```

Comment:
Note that thanks to this call this is done before rendering the component for all tests.